server/fakeData/fakeDataHelpers.js

```diff
@@ -24,11 +24,11 @@ const fillProductTypes = async (n) => {
   for (var i = 0; i < productTypes.length; i++) {
```

Comment:
My only criticism at this point would be the commit message for not being very specific, but I must say I found it quite amusing 😄

Reply:
Thank you. I will try to keep improving my commits messages.